lib/keystores.js

```diff
@@ -198,14 +229,25 @@ api.get = async ({id} = {}) => {
  * @param {string} options.meterId - The ID of the meter to get.
  * @param {object} options.moduleManager - The KMS module manager to use.
  * @param {AbortSignal} [options.signal] - An abort signal to check.
+ * @param {boolean} [options.explain=false] - An optional explain boolean.
  *
- * @returns {Promise<object>} The storage usage for the meter.
+ * @returns {Promise<object> | ExplainObject} The storage usage for the meter
+ *   or an ExplainObject if `explain=true`.
  */
-api.getStorageUsage = async ({meterId, moduleManager, signal} = {}) => {
+api.getStorageUsage = async ({
+  meterId, moduleManager, signal, explain = false
+} = {}) => {
+  const collection = database.collections['kms-keystore'];
+  const query = {'config.meterId': meterId};
+  const projection = {_id: 0, config: 1};
+
+  if(explain) {
+    const cursor = await collection.find(query, {projection});
+    return cursor.explain('executionStats');
+  }
+
   // find all keystores with the given meter ID
-  const cursor = await database.collections['kms-keystore'].find(
-    {'config.meterId': meterId},
-    {projection: {_id: 0, config: 1}});
+  const cursor = await collection.find(query, {projection});
```

Comment:
Since a cursor is used here -- we can just call this once and then move the `explain` conditional down after it. The comment about `// find all keystores...` should come before the query, etc. as it explains why the query is what it is.

```suggestion
  const cursor = await collection.find(query, {projection});
  if(explain) {
    return cursor.explain('executionStats');
  }
```